test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -415,98 +428,74 @@ describe( `[${ host }] Calypso Gutenberg Site Editor Tracking: (${ screenSize })
 
 		createGeneralTests( { it, editorType: 'site', baseContext: 'template' } );
 
-		// Temporarily skip these tests until we can update track events / tests to handle the new
-		// interface.  https://github.com/Automattic/wp-calypso/pull/56544
-		describe.skip( 'Tracks "wpcom_block_editor_global_styles_tab_selected', function () {
-			it( 'when Global Styles sidebar is opened', async function () {
-				const editor = await SiteEditorComponent.Expect( this.driver );
-
-				await editor.toggleGlobalStyles();
-
-				const eventsStack = await getEventsStack( this.driver );
-				const tabSelectedEvents = eventsStack.filter(
-					( [ eventName ] ) => eventName === 'wpcom_block_editor_global_styles_tab_selected'
-				);
-				assert.strictEqual( tabSelectedEvents.length, 1 );
-				const [ , eventData ] = tabSelectedEvents[ 0 ];
-				assert.strictEqual( eventData.tab, GLOBAL_STYLES_ROOT_TAB_NAME );
-				assert.strictEqual( eventData.open, true );
-			} );
-
-			it( 'when Global Styles sidebar is closed', async function () {
-				const editor = await SiteEditorComponent.Expect( this.driver );
-
-				// Note the sidebar is already open here because of the previous test.
-				await editor.toggleGlobalStyles();
+		// Temporarily skip these tests until we can deploy wpcom-block-editor changes from
+		// https://github.com/Automattic/wp-calypso/pull/56551
+		describe.skip( 'Global styles panel events', function () {
```

Comment:
We will have to wait to test these updated tests in a subsequent PR to remove the `.skip` after the `wpcom-block-editor` changes are released.  Currently I cannot get the site editor to load in the e2e test environment when sandboxing widgets, similarly we would not want the tests running and failing after this PR is merged yet before the `wpcom-block-editor` build it released.